src/sentry/utils/appleconnect/itunes_connect.py

```diff
@@ -293,7 +293,7 @@ def set_provider(session: Session, content_provider_id: int) -> None:
         )
     user_id = safe.get_path(user_details_response.json(), "data", "sessionToken", "dsId")
 
-    url = "https://appstoreconnect.apple.com//WebObjects/iTunesConnect.woa/ra/v1/session/webSession"
+    url = "https://appstoreconnect.apple.com//WebObjects/iTunesConnect.woa/ra/v2/session/webSession"
```

Comment:
This updates `set_provider`, but the only use of `set_provider` is currently commented out in `ITunesClient`'s init. Could I be overlooking an invocation somewhere else, or is this updating an unused code path right now?